docs/targets/home-screen.md

```diff
@@ -55,7 +55,25 @@ as a JavaScript array of ``CodeCard`` instance.
 ```
 ````
 
-The ``name``, ``description``, ``url``, ``imageUrl`` are mandatory. They are used to display the card in the home screen. A page with a URL that starts with ``https://`` will automatically open in a new tab.
+The ``name``, ``description``, ``url``, ``imageUrl`` are mandatory. They are used to display the card in the home screen. A page with a URL that starts with ``https://`` will automatically open in a new tab. If you want to have multiple lines in the description, you can
+use an array of strings instead of a single string. The multiple lines are combined into a single paragraph:
```

Comment:
Single paragraph - T or F?

Reply:
Each string in the array would be a separate paragraph

Reply:
Multiple sentences per string or are the truncated at the first period?

Reply:
Please just use \n

Reply:
Looks like you grab just the first element of the split.

Reply:
Okay, fixed to just be support for `\n` rather than ``string | string[]``